packages/garbo/src/combat.ts

```diff
@@ -259,6 +259,14 @@ export class Macro extends StrictMacro {
       .trySingAlong()
       .familiarActions()
       .tryEgg()
+      .externalIf(
+        myFamiliar() === $familiar`Grey Goose` &&
+          $familiar`Grey Goose`.experience >= 36,
```

Comment:
it might be good to make that xp reference an easily edited variable just in case we eventually swap out knights for something else that requires more XP to dupe. but as of now this looks like a good start! thanks seraph 💜

Reply:
We'd need to value the full item array and make some decisions, which should be feasible. We'd also need to do as Rinn suggested and determine whether the full array, or part, or even just the expensive target was dupable which may not be trivial.